src/Components/benchmarkapps/Wasm.Performance/Driver/Wasm.Performance.Driver.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+
+    <OutputType>exe</OutputType>
+
+    <!-- WebDriver is not strong-named, so this test project cannot be strong named either. -->
+    <SignAssembly>false</SignAssembly>
```

Comment:
Don't remember if I mentioned this might break official builds. Probably need to change eng/SignCheckExclusionsFile.txt

Reply:
Or maybe eng/Signing.props Either way, suggest doing an internal build before we merge.

Reply:
Is it sufficient to mark this project as a testasset: `<IsTestAssetProject>true</IsTestAssetProject>`? We do this with the E2E tests and that seems to build fine on the CI

Reply:
I suspect turning off packing will be enough but nothing about `$(IsTestAssetProject)` should hurt